library/src/main/java/me/toptas/fancyshowcase/FancyShowCaseView.java

```diff
@@ -453,6 +454,35 @@ protected void setDismissListener(DismissListener dismissListener) {
         mDismissListener = dismissListener;
     }
 
+    /**
+     * Returns if current shape has a radius. Have to add more cases if new shapes doesn't have radius
+     *
+     * @return true if current shape has a radius
+     */
+    public boolean hasRadius(){
+
+        if(mFocusShape.equals(FocusShape.CIRCLE))
+            return true;
+
+        return false;
+    }
+
+    /**
+     * Returns mRadius or -1 if mRadius == 0 (i.e. Radius doesn't exist for the shape)
+     *
+     * @return mRadius or -1 if mRadius == 0
+     */
+    public int getFocusRadius(){
```

Comment:
Are you sure about this? I've performed several tests on rounded rectangles, and everytime mRadius was == to 175. It's initialized in Calculator line 40 with height and width, enven for ROUNDED_RECTANGLE. So I think if getFocusRadius() has to return -1 for ROUNDED_RECTANGLE, Calculator has to be changed or a test has to be performed in getFocusRadius() no?
Btw ty for your time.